internal/command/kind.go

```diff
@@ -21,18 +21,27 @@ import (
 	"knative.dev/kn-plugin-quickstart/pkg/kind"
 )
 
+var kindNodeImage string
+
 // NewKindCommand implements 'kn quickstart kind' command
 func NewKindCommand() *cobra.Command {
 	var kindCmd = &cobra.Command{
 		Use:   "kind",
 		Short: "Quickstart with Kind",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			fmt.Println("Running Knative Quickstart using Kind")
-			return kind.SetUp(name)
+			return kind.SetUp(name, kindNodeImage)
 		},
 	}
 	// Set kindCmd options
 	clusterNameOption(kindCmd, "knative")
 
+	kindCmd.Flags().StringVar(
+		&kindNodeImage,
+		"image",
+		"kindest/node:v1.23.3",
+		"KinD image name",
+	)
```

Comment:
Could we do this in a function, similar to `clusterNameOption()`?

Reply:
Also, I think we may want to keep the default empty here so that it doesn't override other options, as in #239 (though I'm open to suggestions on better ways to do so)